src/components/executive-education-2u/components/CourseSummaryCard.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Card, Image, Row, Col,
+} from '@edx/paragon';
+import { numberWithPrecision } from '../../course/data/utils';
+
+const CourseSummaryCard = ({ courseMetadata }) => (
+  <Card
+    className="mb-4 course-summary"
+    orientation="horizontal"
+  >
+    <Card.Body>
+      <Card.Header
+        title={(
+          <Image
+            src={courseMetadata.organizationImage}
+          />
+        )}
+      />
+      <Card.Section>
+        <Row>
+          <Col xs={12} lg={{ span: 8, offset: 0 }} className="vertical">
+            <Row>
+              <Col xs={12} lg={{ span: 8, offset: 0 }}>
+                <p className="small font-weight-light text-gray-500">{ courseMetadata.organizationName }</p>
+                <p>{ courseMetadata.title }</p>
+              </Col>
+            </Row>
+          </Col>
+          <Col xs={12} lg={{ span: 4, offset: 0 }}>
+            <div className="course-details">
+              <Row>
+                <Col className="small font-weight-light text-gray-500 justify-content-start">Available start date:</Col>
+                <Col className="justify-content-end"><Row className="justify-content-end margin-right-10">{courseMetadata.startDate}</Row></Col>
+              </Row>
+              <Row>
+                <Col className="small font-weight-light text-gray-500 justify-content-start">Course duration:</Col>
+                <Col className="justify-content-end"><Row className="justify-content-end margin-right-10">{courseMetadata.duration}</Row></Col>
+              </Row>
+              <Row>
+                <Col className="small font-weight-light text-gray-500 justify-content-start">Course total:</Col>
+                <Col className="justify-content-end">
+                  <Row className="justify-content-end margin-right-10">
+                    ${courseMetadata.priceDetails?.price ? `${numberWithPrecision(courseMetadata.priceDetails.price)} ${courseMetadata.priceDetails.currency}` : 'Unknown'}
```

Comment:
Is there are possibility that `courseMetadata.priceDetails?.price` will be `falsy`?

Reply:
I was getting an error for some cases, that is why I added this check.